src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -91,42 +104,43 @@ extends AbstractMap[Key, Value]
     table = new Array[Entry](newSize)
     mask = newSize - 1
     oldTable.foreach( entry =>
```

Comment:
Not related to this PR, but it might be a good idea to replace this with a `while` eventually.